mappings/net/minecraft/class_264.mapping

```diff
@@ -1,6 +0,0 @@
-CLASS net/minecraft/class_264
```

Comment:
Why'd this mapping get deleted? Just because it contains nothing more than some params doesn't mean it should be crabbed

Reply:
Oh that one I thought I had done, I will restore it.